github/repos_test.go

```diff
@@ -457,7 +457,7 @@ func TestRepositoriesService_GetBranch(t *testing.T) {
 
 	want := &Branch{
 		Name:      String("n"),
-		Commit:    &Commit{SHA: String("s")},
+		Commit:    &RepositoryCommit{SHA: String("s")},
```

Comment:
This is good, but I think it's a good opportunity to expand the test data to include a `Commit` inside. That way it'd be a little more representative of the real data you get from GitHub, ala https://developer.github.com/v3/repos/branches/#get-branch.

Something like this:

```Go
want := &Branch{
	Name: String("n"),
	Commit: &RepositoryCommit{
		SHA: String("s"),
		Commit: &Commit{
			Message: String("m"),
		},
	},
	Protected: Bool(true),
}
```

You'd have to update the `mux.HandleFunc("/repos/o/r/branches/b"` above, of course.

Doing this is optional, if you agree it'd be an improvement for this Go package.